lib/screens/contributors.dart

```diff
@@ -11,7 +11,7 @@ import 'package:hacktoberfest_flutter/models/contributors_card_model.dart';
 class Contributors extends StatefulWidget {
   static const String routename = '/Contributors';
 
-  const Contributors({super.key});

```

Comment:
> const Contributors({super.key});

this is absolutely modern way to do this using super keyword
again migrate to newer versions of flutter and dart